R/NEWS.md

```diff
@@ -1,4 +1,13 @@
-# feather (development version)
+# feather 0.4.0
+
+This release updates `feather` to depend on the [`arrow`](https://arrow.apache.org/docs/r/) package, which is where Feather format development has moved. This resolves many bug reports and missing features from the previous implementation of Feather in R, and it brings the R package in line with the Python `feather` package, which has depended on `pyarrow` since 2017.
+
+For compatibility, `feather::write_feather()` uses V1 of the Feather specification: it is a wrapper around `arrow::write_feather(version = 1)`. Feather V2 is just the Arrow format on disk and has support for a much richer set of data types. To switch to V2, we recommend just using `arrow::write_feather()`.
```

Comment:
Do we want to encourage folks to move to V2? Or just show that one can?

Reply:
If we're going to encourage V2 more heavily, we should encourage people just to use arrow and not bother with feather, there's no value added. And actually we make that recommendation in the README. So I think this is fine, but feel free to suggest new words here.